modules/hostap/CMakeLists.txt

```diff
@@ -285,7 +285,6 @@ zephyr_library_sources_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_WPS
 	${HOSTAP_SRC_BASE}/wps/wps_enrollee.c
 	${HOSTAP_SRC_BASE}/wps/wps_registrar.c
 	${HOSTAP_SRC_BASE}/crypto/dh_groups.c
-	${HOSTAP_SRC_BASE}/crypto/dh_group5.c
```

Comment:
How is this change related to WPS?

Reply:
The reason why delete dh_group5.c is multiple definition compile errors when enable CONFIG_WIFI_NM_WPA_SUPPLICANT_WPS.
Actually, the I don't know the root reason why need use dh_groups.c but not dh_group5.c, the below 2 points support changed:
A. WPA3 also use  dh_groups.c [CONFIG_WIFI_NM_WPA_SUPPLICANT_WPA3].
B. I think dh_group5.c and dh_groups.c both are APIs support for ECDH,  dh_groups.c works ok for WPA3 SAE, I guess it should also will works ok for WPS.

Reply:
Independent of the reasoning, this is an independent change, so, has to be in a separate commit with commit log explaining the reason.

Reply:
Okay, if no build errors reported by CI of this pr after remove this change, will raise separate commit for it